### ISSUE TITLE
Update use of Input Placeholders to support client theme overrides

### ISSUE THREAD

**Issue by bryancasler** [4site-interactive-studios/engrid-scripts#153]
Currently engrid-scripts sets its own field placeholders. Then each client theme that wants something different defines its own JS to override it. This causes the engrid-scripts placeholders to populate and then a moment later (but still long enough to be visible) the client theme's JS fires and replaces engrid-scripts placeholders. We should create a system by which the client theme can define overrides and then placeholders are only populated once those overrides replace the defaults, populating inputs with placeholders only once.

**Comment by bryancasler** [comment]
I don't think this was done, each client theme looks to be setting them their own way.

**Comment by MichaelT372** [comment]
I think the option is in ENgrid "Placeholders". It may be a case that client themes need updating to use this instead of their old way of doing it. Add 5 hours to investigate and fix client themes.
